sql/parser/datum.go

```diff
@@ -333,10 +333,9 @@ func (d DBytes) String() string {
 	return encodeSQLBytes(string(d))
 }
 
-// DDate is the date Datum.
-type DDate struct {
-	time.Time // Must always be UTC!
-}
+// DDate is the date Datum represented as the number of days after
+// the Unix epoch.
+type DDate int64
```

Comment:
possibly pedantic: do we want to make this a struct containing an int64 so that it can't just be cast?